app/scripts/dimApp.config.js

```diff
@@ -77,12 +76,13 @@
         $window.initgapi = function() {
           SyncService.init();
         }
+
         SyncService.get().then(function(data) {
-          data = (data) ? data['2016.03.13-v3.4.0'] : null;          
-          if(_.isNull(data) || _.isEmpty(data)) {
+          var toastViewedFlag = data['2016.03.13-v3.4.0'] || {};
+          if(_.isNull(toastViewedFlag)) {
```

Comment:
This was null originally. I think I changed it back as part of some smoke tests. I've reverted it to null and wrapped the entire block in a null check for the data variable as it was throwing console errors